### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by mruminski**

### CODE REVIEW

File: script-compiled.js [hunk 1]
```diff
@@ -1 +1,47 @@
-"use strict";
```
Comment (matwjablonski) [inline script-compiled.js:1]: this file is compiled, so you shoudn't add it to repository

File: script.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+function say() {
+  console.log(`${hello}, ${world}`);
+}
+
+say();
+
+const multiply = (x = 1 ,y = 1) => { console.log(x * y) };
+
+multiply(2,2);
```
Comment (matwjablonski) [inline script.js:12]: ad space after comma

File: script.js [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+multiply(5);
+
+function average(...args) {
+  const len = args.length;
+  return args.reduce((prev, curr) => {
```
Comment (matwjablonski) [inline script.js]: try to write is without {} brackets and without assigning it to new variable

Comment (matwjablonski) [reply script.js]: what is second argument of reduce function?

Comment (mruminski) [reply script.js]: a current value

File: script.js [hunk 4]
```diff
@@ -33,3 +29,22 @@ const firstname = third;
...
+  { name: 'Tomek', amount: 10 },
+  { name: 'Alicja', amount: 15 },
+  { name: 'Krzysztofa', amount: 20 },
+  { name: 'Madzia', amount: 25 },
+  { name: 'Zbyszek', amount: 30 }
+];
+
+const initBottleTops = 27;
+
+function sumOfTops() {
```
Comment (matwjablonski) [inline script.js]: use arrow functions ;)

**Review by matwjablonski — APPROVED** [review]

**Pull request merged**